components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/mappings/APIMappingUtil.java

```diff
@@ -82,8 +82,8 @@ public static APIDTO toAPIDTO(API api) {
         apiDTO.setVersion(api.getVersion());
         apiDTO.setContext(api.getContext());
         apiDTO.setDescription(api.getDescription());
-        apiDTO.setPolicies(api.getPolicies());
-        apiDTO.setLabels(api.getLabels());
+        apiDTO.setPolicies(new ArrayList<>(api.getPolicies()));
```

Comment:
okay understood. Since we don't have a way to have "Set" concept in swagger, this makes sense.